settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Słaba nazwa pliku...
Bez patrzenia do środka nie wpadniesz że to są settingsy moch-y.
Co powiesz na mocha-settings.json?

Reply:
bo to teoretycznie nie są settingsy mochy, tylko całego projektu, a że musiałem zmienić tylko ustawienia mochy to tak wyszło